### PR TITLE
New pronouns and display_name fields on User object

### PR DESCRIPTION
**Pull request by elloboblanco**
Added documentation about new `pronouns` and `display_name` fields on the User object. While I was in here I also trimmed down a bunch of the supplemental data objects to make our examples easier to digest (although they are not valid JSON now).

### CODE REVIEW

File: source/includes/APIReference/ManagedClients/Examples/_managing-response.md.erb [hunk 1]
```diff
@@ -1,71 +1,15 @@
...
-          "manage_schedules": true,
-          "manage_company_schedules": false,
-          "view_my_schedules": false,
-          "view_group_schedules": false,
-          "view_company_schedules": false
+    "results": {
+        "managed_clients": {
+            "1234": {
+                "id": "1234",
+                "company_url": "fbi",
```
Comment (dustinsweet) [inline source/includes/APIReference/ManagedClients/Examples/_managing-response.md.erb]: Lol.  Funny, but I think you should change it.

File: source/includes/APIReference/Users/Examples/_create-response.md.erb [hunk 2]
```diff
@@ -24,6 +24,8 @@
         "client_url": "api_sample_output",
         "company_name": "API Sample Output Company",
         "profile_image_url": "",
+        "display_name": null,
```
Comment (dustinsweet) [inline source/includes/APIReference/Users/Examples/_create-response.md.erb:27]: I notice all of the examples have nulls for both new properties.  Any particular reason we're not using example values?

Comment (elloboblanco) [reply source/includes/APIReference/Users/Examples/_create-response.md.erb:27]: Field is null for everyone until the feature is enabled (which could be in a while). The docs on the User Object page describes this.

File: source/includes/APIReference/ManagedClients/Examples/_managing-response.md.erb (hunk 1 above)
Comment (elloboblanco) [reply source/includes/APIReference/ManagedClients/Examples/_managing-response.md.erb]: Updated!

**Review by dustinsweet — APPROVED** [review]

**Pull request merged**